src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
Instead of checking for `null` here why not have `addFailure()` throw if it is passed a `null`? Seems like a programmer error for the caller to add a `null`. Currently any caller of `getFailures()` has to check for `null` values.